### DISCUSSION TITLE
Strange glitch

### DISCUSSION THREAD

**Discussion by MikleBas** [wavelog/wavelog#1200] [General]
Today I wanted to call a station, I enter its callsign in the window, well, the result can be seen in the screenshot
it shows that I already had QSO with him, although it is clear that there are no matches      ¯\_(ツ)_/¯

![Screenshot](https://github.com/user-attachments/assets/fed93433-0841-4eee-baed-56386c457206)

**Comment by AndreasK79** [comment]
Yeah, I can reproduce that problem here too. It's because of the double suffix. If you just log R4FBJ/0, it would work normally. I will take a look to see if it's possible to do some adjustments.

**Comment by AndreasK79** [comment]
I am closing this discussion, as I made an issue instead.
